app/assets/javascripts/application.js

```diff
@@ -62,6 +62,9 @@
 = require 'classnames'
 = require 'react-input-autosize'
 = require 'react-select'
+= require 'moment'
+= require 'daypicker'
```

Comment:
I think you can also pull in the daypicker using the rails-assets tool: https://rails-assets.org  That lets you bring any bower package into rails. We are using this in LARA, and daypicker seems to be available in bower.